iceprod/server/data/www/rest.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Helper functions that use REST API
+ */
+
+async function fetch_json(url, passkey, json=null) {
+	try {
+		var payload = {headers: new Headers({'Authorization': passkey})};
```

Comment:
The header should be of the form `Authorization: Bearer <token>`

Reply:
As in `{'Authorization': 'Bearer ' + passkey}`?

Reply:
yes

Reply:
5977009